Chandra/cmd_states/cmd_states.py

```diff
@@ -107,6 +108,23 @@ def add_trans(date=date, **kwargs):
     return add_trans
 
 
+def _make_pitch_cmds(cmds, sample_time=5000):
+    """
+    Add cmds to clock out pitch for any long states
+    """
+    cmd_times = np.array(DateTime([cmd['date'] for cmd in cmds]).secs)
+    durations = cmd_times[1:] - cmd_times[0:-1]
+    long_dur = durations > sample_time
```

Comment:
It would be cleaner to just clock out these pitch requests at equal intervals over the range of cmds, but that could cause conflicts at the maneuvers.  I *think* that just using these long durations should be enough of a filter to prevent that problem.  Not tested.